api_calls/reference/special_offers/create-request.json

```diff
@@ -1,9 +1,7 @@
 {
   "special_offers": [
     {
-      "name":{
-        "en":"Special offer for 1"
-      },
```

Comment:
Is this approach not considered **on purpose**?
I think it looks pretty handy **if** we are intending to support multiple languages here in the future

Reply:
indeed it would be usefull, but for now it doesn't work as docs describe, so it's confusing